include/oneapi/dpl/pstl/hetero/dpcpp/unseq_backend_sycl.h

```diff
@@ -485,23 +479,21 @@ struct reduce_over_sub_group
  
     template <typename _NDItemId, typename _Size, typename _AccLocal>
     inline _Tp
-    reduce_impl(const _NDItemId& __item_id, const _Size& __n, const _AccLocal& __local_mem,
+    reduce_impl(const _NDItemId& __item_id, const _Size& __n, const _Tp& __val, const _AccLocal& __local_mem,
```

Comment:
Isn't the `__local_mem` parameter unneeded now?

Reply:
Ah, I guess maybe you left it so that all possible `__reduce_pattern`s have the same call signature? I think that's fine for now but  maybe we should leave a `TODO:` to refactor that slightly?

Reply:
Done, iv removed the local accessor